### PR TITLE
chore: add bun 1.0 post

### PR DESCRIPTION
**Pull request by hendriknielaender**
as a first draft

### CODE REVIEW

File: data/blog/bun-1-0.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+imageCreditUrl: https://labs.openai.com/
+tags: [gptpost, nodejs, bun, typescript, javascript, release]
+tweet: "https://twitter.com/doubletrblblogs/status/1680501484037844992"
+---
+
+## Introduction
+With the fast-paced evolution in the tech world, developers often find themselves torn between sticking to what they know and venturing into the promise of something new. While Node.js has held its own for a long time, a new contender, Bun, has recently stepped onto the scene, suggesting a future that many may not have anticipated.
+
+Bun vs. Node.js: The Subtle Revolution\
+When change is brought up in tech discussions, it often echoes innovations that revolutionize the way we operate. However, sometimes the most profound changes are those that seamlessly integrate into our current workflows.
```
Comment (flyck) [inline data/blog/bun-1-0.md:15]: > When change is brought up in tech discussions, it often echoes innovations that revolutionize the way we operate.

bruh

**Comment by flyck** [comment]
image

**Comment by hendriknielaender** [comment]
The one with the light blue background are going into the right direction, but maybe multiple bun's on a baking tray (then we could add the monorepo example)? 😅 

image

**Comment by flyck** [comment]
> The one with the light blue background are going into the right direction, but maybe multiple bun's on a baking tray (then we could add the monorepo example)? 😅
> 
> image

what do you see that is left for acme-corp bun? maybe that could even derserve its own post if its more work 🤔 

alternatively we can also repurpose the serverless bun lambda into one without the jimp stuff. that package has weird exports.. maybe just read a sqs message and write it into a dynamodb 🤔

**Comment by flyck** [comment]
Bun 1.0 post discarded

**Pull request closed without merging**